doc/source/whatsnew/v1.3.0.rst

```diff
@@ -645,7 +645,7 @@ If installed, we now require:
 +-----------------+-----------------+----------+---------+
 | mypy (dev)      | 0.812           |          |    X    |
 +-----------------+-----------------+----------+---------+
-| setuptools      | 38.6.0          |          |    X    |
+| setuptools      | 51.0.0          |          |    X    |
```

Comment:
is this for users building pandas from source or users using a pre-built pandas with matplotlib?